inventories/README.md

```diff
@@ -2,17 +2,49 @@
 <!-- Copyright (C) 2024, SFL (https://savoirfairelinux.com) -->
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
 
+# Ansible configuration
+Configuring the cluster through ansible requires inventories and we recommand 3 kinds:
+- The description of the machines that will form the cluster
+- The description of the OpenVSwitch network
+- The description of the virtual machines
+In addition to this a file to configure the firewall rules for the cluster's network is needed
+
 # Inventories directory
 
 This is the place where you can store your inventories.
-We recommend to have 3 kinds of inventories:
-- The cluster description inventory, where you will populate the details of your nodes (ip, interface, network details, disks, etc.): a template is provided (seapath_cluster_definition_example.yml)
-- The OpenVSwitch topology inventory, where you will describe the different bridges and ports configuration for OVS: a template is provided (seapath_ovstopology_definition_example.yml)
-- The VM inventory, where you will describe all variables related to the virtual machine (number of cores, features, ip, network interface, etc.): a template is provided (seapath_vm_definition_example.yml)
+In this folder, different templates are provided:
+- The cluster description inventory, where you will populate the details of your nodes (ip, interface, network details, disks, etc.): seapath_cluster_definition_example.yml
```

Comment:
I recommend to put each file name in a code beacon: `seapath_cluster_definition_example.yml`

Reply:
Corrected by last commit

Reply:
I don't see it on my side